tests/cypress/e2e/copyCutPaste.cy.ts

```diff
@@ -0,0 +1,95 @@
+import { GraphqlUtils } from '../utils/graphqlUtils'
+import { PageComposer } from '../page-object/pageComposer'
+
+const checkDescriptions = (path: string) => {
+    GraphqlUtils.getNode(path, 'jcr:description', 'en', "That's the description")
+    GraphqlUtils.getNode(path, 'jcr:description', 'fr', "C'est la description")
+}
+
+describe('Copy Cut and Paste tests with page composer', () => {
+    before('Create required content', () => {
+        cy.login()
+        cy.executeGroovy('createSite.groovy', { SITEKEY: 'testsite' })
+        GraphqlUtils.setProperty('/sites/digitall/home/about', 'jcr:description', "That's the description", 'en')
+        GraphqlUtils.setProperty('/sites/digitall/home/about', 'jcr:description', "C'est la description", 'fr')
+        GraphqlUtils.addVanityUrl('/sites/digitall/home/about', 'en', '/about')
+        cy.logout()
+    })
+
+    it('Copy and paste in another site', () => {
+        const composer = new PageComposer()
+        cy.login()
```

Comment:
I know, it's not specified in the ticket, but shouldn't it be better to execute the tests with a editor instead of the root user